packages/js/components/src/progress-bar/index.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+
+import { HTMLAttributes, createElement } from 'react';
+/**
+ * Internal dependencies
+ */
+
+type ProgressBarProps = {
+	className?: string;
+	percent?: number;
+	color?: string;
+	bgcolor?: string;
+};
+
+export const ProgressBar = ( {
```

Comment:
Would it be clearer to name this HeaderProgressBar? (thanks ChatGPT, lol)

Reply:
Since this component is really basic and can be used for other places, `ProgressBar` makes more sense IMO. HeaderProgressBar sounds a bit too specific although that's how we're using it :)